src/components/Footer.cy-spec.js

```diff
@@ -0,0 +1,63 @@
+// compare to tests in "Footer.spec.js"
+import React from 'react'
+import Footer from './Footer'
```

Comment:
note we only import Footer - cause we don't care what children components it uses
compare to https://github.com/cypress-io/cypress-example-todomvc-redux/blob/master/src/components/Footer.spec.js